openmc/mgxs_library.py

```diff
@@ -1634,6 +1685,267 @@ def set_inverse_velocity_mgxs(self, inverse_velocity, temperature=294.,
         self._inverse_velocity[i] = inverse_velocity.get_xs(
             nuclides=nuclide, xs_type=xs_type, subdomains=subdomain)
 
+    def convert_representation(self, target_representation, num_polar=None,
+                               num_azimuthal=None):
+        """Produce a new XSdata object with the same data, but converted to the
+        new representation
```

Comment:
well it can be either angular or isotropic and this refers to the MGXS representation not really the scattering format.  either way I clarified a little bit.

- [x]